### DISCUSSION TITLE
Fabric API 1.17.1 can't run

### DISCUSSION THREAD

**Discussion by Riiunn** [FabricMC/fabric-api#1568] [Player support]
I'm trying to update my fabric API to 1.17.1 but whenever I look at it in my mods folder it says its 1.17. Also whenever I try to run Minecraft with fabric I get an error message that I've never seen before. I can run fabric without the API but with it I cant. Any help on what to do? 
![2021-07-14 (2)](https://user-images.githubusercontent.com/87435399/125646561-0c5ebfdb-435d-4e84-b06e-c37e13cf8133.png)

**Comment by haykam821** [comment]
If you are running Fabric with Minecraft **1.17**, you should use version [`0.36.0+1.17`](https://github.com/FabricMC/fabric/releases/tag/0.36.0%2B1.17) of Fabric API. This is the last version of Fabric API that will support the original Minecraft 1.17.

If you are running Fabric with Minecraft **1.17.1**, you should use a later version of Fabric API (for example, [`0.37.0+1.17`](https://github.com/FabricMC/fabric/releases/tag/0.37.0%2B1.17)).

**Reply by LunarKoala** [reply]
it does the same thing with that version of fabric API when I try to download it
-You can just change the name of the file to 1.17.1
